runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -249,6 +259,10 @@ public void render(Graphics2D graphics)
 						case TOP_RIGHT:
 							topRightPoint.y += dimension.height + (dimension.height == 0 ? 0 : PADDING);
 							break;
+						case ABOVE_CHATBOX_RIGHT:
+							rightChatboxPoint.x -= dimension.width - (dimension.width == 0 ? 0 : PADDING);
```

Comment:
Moving both x and y will make any additional overlay move diagonaly top-left in the ABOVE_CHATBOX_RIGHT area, move either x or y here (y would be better to not clash with BOTTOM_RIGHT, so the overlays will stack above each other in ABOVE_CHATBOX_RIGHT

Reply:
dimension.width + (dimension.width == 0